dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -154,7 +154,7 @@
             String textValue = UtilMethods.isSet(value) ? value.toString() : (UtilMethods.isSet(defaultValue) ? defaultValue : "''");
             %>
             <script src="/html/dotcms-block-editor.js"></script>
-            <dotcms-block-editor style="width: 100%; height: 500px; display: block;"></dotcms-block-editor>
```

Comment:
We do want a max-height here, so this is not a fix, we need to figure it out how we can do a max-height without breaking the editor.